substrate-query-framework/cli/src/templates/entities/resolver.ts.mst

```diff
@@ -99,12 +99,8 @@ export class {{className}}Resolver {
 
   @Query(() => {{className}}, { nullable: true })
   async {{camelName}}(@Arg('where') where: {{className}}WhereUniqueInput): Promise<{{className}} | null> {
-    try {
-      return await this.service.findOne<{{className}}WhereUniqueInput>(where);
-    } catch (error) {
-      if (error.message.includes('Unable to find')) return null;
-      throw error;
-    }
+    const result = await this.service.manager.findOne({{className}}, where);
```

Comment:
It's better to fetch only fields requested from the API. We can use `@Fields() fields: string[]` to get the requested fields. So we may instead do
```
const result = await this.service.find<{{className}}WhereUniqueInput>(where, null, 1, 0, fields); //where, orderBy, limit, offset, fields
if (result && result.length >= 1) {
   return result[0]
}
return null
```

Reply:
thanks! it's better, done.